index.bs

```diff
@@ -3056,9 +3056,12 @@ To <dfn>process an attribution source</dfn> given an [=attribution source=] |sou
 Note: Because a fake report does not have a "real" effective destination, we need to subtract from the
 privacy budget of all possible destinations.
 
-Note: The limits that are not reported as <code>[=source debug data type/source-success=]</code> should be
-checked before any limits that are reported implicitly as <code>[=source debug data type/source-success=]</code>
-to prevent side-channel leakage of cross-origin data. Furthermore, the [=verbose debug data=]
+Note: The limits that are not reported as <code>[=source debug data type/source-success=]</code>
+in [=verbose debug reports=] should be checked before any limits that are reported implicitly as
+<code>[=source debug data type/source-success=]</code> (including
```

Comment:
Does "including" in this case mean that there are others? Or is the list exhaustive?

Reply:
The list is exhaustive, I can remove `including` if it's confusing.

Reply:
Sounds good.

Reply:
Removed `including`, thanks.